### PR TITLE
TINY-9911: Changed gulp-file to generate javascript files instead of css.

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket: 

Description of Changes:
Used for testing withing the epic. Not intended for production release at this time. Will not pass tests due to removing the css files.

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

### CODE REVIEW

File: modules/oxide/gulpfile.js [hunk 1]
```diff
@@ -90,7 +92,14 @@ gulp.task('minifyCss', function() {
   return gulp.src(['./build/skins/**/*.css', '!**/*.min.css'])
     .pipe(sourcemaps.init())
     .pipe(cleanCSS({ rebase: false }))
-    .pipe(rename({ extname: '.min.css' }))
+    .pipe(through2.obj(function(file, _, cb) {
+      if (file.isBuffer()) {
+        const contents = `tinymce.resources.add('${file.relative}', '${file.contents.toString().replace(/'/g, "\\'")}')`;
```
Comment (spocke) [inline modules/oxide/gulpfile.js:97]: I think it's better to use JSON.stringify here so make sure that all things gets escaped properly.

Comment (HAFRMO) [reply modules/oxide/gulpfile.js:97]: Which part do you want to put stringify on? The file is css, so it shouldn't have anything to escape in the first place?

Comment (spocke) [reply modules/oxide/gulpfile.js:97]: CSS has plenty of things that might need escaping.

All this is valid css:
```
[foo="bar"], [foo='bar'] {}
.foo { background: url('bar') };
.foo { background: url("bar") };
```

Comment (spocke) [reply modules/oxide/gulpfile.js:97]: Might be other characters in there as well. Just replacing the quotes just seems a bit weak that all. JSON.stringify will make sure that it's always a valid string.

Comment (HAFRMO) [reply modules/oxide/gulpfile.js:97]: Alright. Can do that.

**Review by vpyshnenko — APPROVED** [review]

**Pull request merged**